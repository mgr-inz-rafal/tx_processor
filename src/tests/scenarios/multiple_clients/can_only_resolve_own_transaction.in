type,client,tx,amount
deposit,1,1,1
deposit,1,2,2
deposit,1,3,3
deposit,1,4,4
deposit,2,5,10
deposit,2,6,20
deposit,2,7,30
deposit,2,8,40
dispute,1,1,
dispute,1,5,
dispute,1,2,
dispute,1,6,
resolve,1,1,
resolve,1,5,
chargeback,1,6,
chargeback,1,2,