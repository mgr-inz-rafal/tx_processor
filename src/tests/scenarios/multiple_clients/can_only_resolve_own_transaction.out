client,available,held,total,locked
1,8,0,8,true
2,100,0,100,false

